geoportailv3/templates/index.html

```diff
@@ -397,6 +404,7 @@ <h2 translate>Feedback</h2>
          appModule.value('ngeoLayertreeTemplateUrl', 'catalog/layertree.html');
          appModule.value('ngeoPopupTemplateUrl', 'layerinfo/popup.html');
          appModule.value('ngeoScaleselectorTemplateUrl', 'infobar/scaleselector.html');
+         appModule.value('ngeoOfflineTemplateUrl', function() {return '../node_modules/ngeo/src/offline/component.html'});
```

Comment:
The path should be handled by pyramid. See how we did it in the 3D branch: https://github.com/Geoportail-Luxembourg/geoportailv3/blob/3d/geoportailv3/templates/index.html#L405.